pages/admin/AdminHandler.php

```diff
@@ -274,27 +265,20 @@ private function siteSettingsAvailability($request)
             'siteAppearanceSetup',
             'statistics',
             'announcements',
-        ];
-
-        if (!Config::getVar('features', 'site_announcements')) {
-            $tabs = array_filter(
-                $tabs,
-                function($tab) { return $tab !== 'announcements'; }
-            );
-        }
-
-        $singleContextSite = (Services::get('context')->getCount() == 1);
+        ], true);
 
-        $tabsAvailability = [];
+        $tabs['announcements'] = (bool) Config::getVar('features', 'site_announcements');
+        $tabs['highlights'] = (bool) Config::getVar('features', 'highlights');
 
-        foreach ($tabs as $tab) {
-            $tabsAvailability[$tab] = true;
-            if ($singleContextSite && !in_array($tab, $tabsSingleContextAvailability)) {
-                $tabsAvailability[$tab] = false;
+        $isSingleContextSite = Services::get('context')->getCount() < 2;
+        if ($isSingleContextSite) {
+            $multipleContextTabs = ['siteSetup', 'languages', 'bulkEmails', 'statistics'];
```

Comment:
I think this is wrong -- those are single context tabs

Reply:
Yes! Looks like I had a brief moment of dyslexia and inverted things 😁